### ISSUE TITLE
Any chance of porting this to Visual Studio code?

### ISSUE THREAD

**Issue by Nexus7** [gmtDevs/atom-ethereum-interface#29]
It would be great if there was a port so that this could work with the VS Code editor... Any chance of that?

**Comment by 0mkara** [comment]
I think VS already has some good plugin for ethereum. I am a complete linux person. Can't say if this can be ported to VS.

**Comment by Nexus7** [comment]
Hey, thanks for the reply... actually VS Code runs on all platforms. I am using it on linux. It is different to the full visual studio and a lot more like atom. There is an existing ethereum plugin but it only does compilation and not deploying or function calling... It would be great to have those features too :-)

**Comment by 0mkara** [comment]
Ohh cool. Can you please link me to the plugin ? I will see what I can do.

**Comment by Nexus7** [comment]
This is the current plugin I use in VS Code: https://github.com/juanfranblanco/vscode-solidity

As you can see it is a little limited to compilation only at the moment...

**Comment by mallapurbharat** [comment]
I agree with Nexus7... It'd be awesome if this could be used with VS Code editor. I use VS Code on my Ubuntu 16.04 machine, and it's probably the most popular editor out there right now IMHO.

I installed atom just for your etheratom package, so it'd be nice if you could add this to VS Code as well.
